src/mappings/roundsManager.ts

```diff
@@ -177,6 +179,59 @@ export function newRound(event: NewRound): void {
     }
   }
 
+  // Update rolling fee windows for tracked broadcasters (gateways)
+  let trackedBroadcasters = protocol.activeBroadcasters;
+  let activeBroadcasters: string[] = [];
+  if (trackedBroadcasters && trackedBroadcasters.length) {
```

Comment:
I believe this can be changed to `if (trackedBroadcasters.length) { ... }` as `trackedBroadcasters` is a non-nullable field.